src/screens/DiscoveryScreenV2/elements/DiscoveryTab.js

```diff
@@ -94,7 +94,11 @@ const styles = StyleSheet.create({
     textAlign: 'center',
     paddingVertical: 10
   },
-  underlineFocus: {
+  underlineInactive: {
+    borderBottomColor: COLORS.gray200,
+    borderBottomWidth: 2
+  },
+  underlineActive: {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `underlineInactive` style and the renaming of `underlineFocus` to `underlineActive` improve readability and make the code more self-explanatory. Ensure that the colors used (`COLORS.gray200` for inactive and `COLORS.signed_primary` for active) align with your new color scheme.
```diff
-  underlineFocus: {
+  underlineInactive: {
+    borderBottomColor: COLORS.gray200,
+    borderBottomWidth: 2
+  },
+  underlineActive: {
     borderBottomColor: COLORS.signed_primary,
     borderBottomWidth: 2
   }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->